src/middleware/get-missing-headers.ts

```diff
@@ -1,4 +1,4 @@
-module.exports = getMissingHeaders;
+import { Request } from "express";
```

Comment:
Why importing Request from "express"? This is not an express middleware, but a Node.js middleware. The two are compatible. You can 

```
import { IncomingMessage} from "http"
```

and then do

```ts
request: typeof IncomingMessage
```

Reply:
> import { IncomingMessage} from "http"

It's what I tried on first instance but I got type issues from here:
https://github.com/octokit/webhooks.js/blob/d675137b32c5867c3fe5f1d204a8075b85cb83d1/src/middleware/get-payload.js#L4-L8

![image](https://user-images.githubusercontent.com/2574275/79703814-59c19280-82ae-11ea-928e-b1c5c0042b75.png)

For the rest of places, the type it's fine. Checking the PR of those lines, seems to be relying on body property. Researching a bit the origin of `body-parse` I ended up finding express types as origin (and those are an extension from `IncomingMessage` of `http`)

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/069f404e6611098c620a265d05782db36efb4fc6/types/express/index.d.ts#L22

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/069f404e6611098c620a265d05782db36efb4fc6/types/express-serve-static-core/index.d.ts#L216

---

How should we proceed here? 
* Apply everywhere ` { IncomingMesage } from 'http'` and `{ Request } from 'express'` for `src/middleware/get-payload.ts`
* Apply everywhere ` { IncomingMesage } and remove if `request.body` since it won't be possible to happen anymore due to type restrictions
* Keep `express` types everywhere

Reply:
ping @gr2m :)